docs/production/READMEs/extended_genome_aligner.rst

```diff
@@ -45,7 +45,7 @@ in bash
 Update the master database
 --------------------------
 
-The pipeline requires a "master" database. This is a compara database containing information that is required to maintain consistency across several production and release databases. See README-master_database for details on how to create an initial master database. 
+The pipeline requires a "master" database. This is a compara database containing information that is required to maintain consistency across several production and release databases. See README-master_database for details on how to create an initial master database.
```

Comment:
So it doesn't look like you simply removed a trailing white space (and while we are here):
```suggestion
This pipeline requires a compara "master" database containing information that maintains consistency across several production and release databases. See README-master_database for details on how to create an initial master database.
```